src/charm.py

```diff
@@ -146,6 +162,23 @@ def _on_tracing_relation_changed(self, e: RelationEvent):
         if not self._tracing.is_v2(e.relation):
             self._publish_v1_data(e.relation)
 
+    def _on_ingress_relation_created(self, e: RelationEvent):
+        if self._ingress.is_ready():
+            self._ingress.submit_to_traefik(
+                self._ingress_config, static=self._static_ingress_config
+            )
+
+    def _on_ingress_relation_joined(self, e: RelationEvent):
+        self._ingress._relation = e.relation
```

Comment:
what's going on here?

Reply:
You were right in another conversation - this looks like a Harness issue inspired by grafana-k8s-operator workaround. Rewrote the unit test to be a scenario test for the same test case and removed the line as it passes.